apps/designer/remix.config.js

```diff
@@ -9,10 +9,7 @@ module.exports = {
   // server. This does not understand the vercel lambda module format,
   // so we default back to the standard build output.
   server: process.env.NODE_ENV === "development" ? undefined : "./server.js",
-  // This makes sure remix bundles monorepo packages which are always
-  // sym-linked and can't be hoisted. You could also manually name packages
-  // e.g. ["@webstudio-is/ui", ...]
```

Comment:
I didn't see a strong reason to bundle them. Monorepo packages should behave like regular packages - for the most part. 

Removing them from there fixes bundling issues around the `sharp` module. This module **can't** be bundled.

Reply:
sharp should be only used serverside ever

Reply:
yes, and it was - but after my changes, it started to be bundled by Remix (whereas on the main branch `require('sharp')` is left in the bundle). I'm not exactly sure why that has changed - it's probably something related to how Remix/esbuild interpret externals or to how Remix uses this `serverDependenciesToBundle`. By bundling sharp we screwed up a relative require contained in that package:
https://github.com/lovell/sharp/blob/9fa516e8496d96fc12a4cba25216e467e36d62b7/lib/sharp.js#L7